script/mail_new_activity.rb

```diff
@@ -66,7 +66,7 @@ def story_subject(story, prefix = "")
 
   mailing_list_users = User.where("mailing_list_mode > 0").select(&:is_active?)
 
-  last_story_id = (Keystore.value_for(LAST_STORY_KEY) || Story.last.id).to_i
+  last_story_id = (Keystore.value_for(LAST_STORY_KEY) || Story.last && Story.last.id).to_i
```

Comment:
I think you can use the safe navigation operator here http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

```suggestion
  last_story_id = (Keystore.value_for(LAST_STORY_KEY) || Story.last&.id).to_i
```

Reply:
We have a custom rubocop linter preventing it.